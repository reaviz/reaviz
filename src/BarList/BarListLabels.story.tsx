import React from 'react';
import { BarList } from './BarList';
import { BarListSeries } from './BarListSeries';

export default {
  tags: ['snapshot'],
  title: 'Charts/Bar List/Labels',
  component: BarList
};

export const LabelStart = () => (
  <BarList
    id="label-start"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries labelPosition="start" valuePosition="end" />}
  />
);

export const LabelBottom = () => (
  <BarList
    id="label-bottom"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries labelPosition="bottom" valuePosition="end" />}
  />
);

export const LabelEnd = () => (
  <BarList
    id="label-end"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries labelPosition="end" valuePosition="start" />}
  />
);

export const LabelTop = () => (
  <BarList
    id="label-top"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries labelPosition="top" />}
  />
);

export const LabelNone = () => (
  <BarList
    id="label-none"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries labelPosition="none" valuePosition="end" />}
  />
);

export const ValueBottom = () => (
  <BarList
    id="value-bottom"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries valuePosition="bottom" />}
  />
);

export const ValueEnd = () => (
  <BarList
    id="value-end"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 100 },
      { key: 'Critical Failure', data: 55 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 200 }
    ]}
    series={<BarListSeries valuePosition="end" />}
  />
);

export const ValueStart = () => (
  <BarList
    id="value-start"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries valuePosition="start" />}
  />
);

export const ValueNone = () => (
  <BarList
    id="value-none"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 50 },
      { key: 'Critical Failure', data: 25 },
      { key: 'Physical Intrusion', data: 5 },
      { key: 'Phishing Attempts', data: 100 }
    ]}
    series={<BarListSeries valuePosition="none" />}
  />
);

export const LongValueStart = () => (
  <BarList
    id="long-value-start"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 5000 },
      { key: 'Critical Failure', data: 50000 },
      { key: 'Physical Intrusion', data: 500000 },
      { key: 'Phishing Attempts', data: 5000000 }
    ]}
    series={<BarListSeries valuePosition="start" />}
  />
);

export const LongValueEnd = () => (
  <BarList
    id="long-value-end"
    style={{ width: 350 }}
    data={[
      { key: 'Vulnerability Patch', data: 5000 },
      { key: 'Critical Failure', data: 50000 },
      { key: 'Physical Intrusion', data: 500000 },
      { key: 'Phishing Attempts', data: 5000000 }
    ]}
    series={<BarListSeries valuePosition="end" />}
  />
);
