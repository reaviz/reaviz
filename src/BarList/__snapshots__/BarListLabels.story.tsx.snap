// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Charts/Bar List/Labels LabelBottom smoke-test 1`] = `
<section role="list"
         id="label-bottom"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelBottom_18e6r_15"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelBottom_18e6r_15"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelBottom_18e6r_15"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelBottom_18e6r_15"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels LabelEnd smoke-test 1`] = `
<section role="list"
         id="label-end"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelEnd_18e6r_35"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelEnd_18e6r_35"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelEnd_18e6r_35"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelEnd_18e6r_35"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels LabelNone smoke-test 1`] = `
<section role="list"
         id="label-none"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelNone_18e6r_24"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelNone_18e6r_24"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelNone_18e6r_24"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelNone_18e6r_24"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels LabelStart smoke-test 1`] = `
<section role="list"
         id="label-start"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelStart_18e6r_8"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelStart_18e6r_8"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelStart_18e6r_8"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelStart_18e6r_8"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels LabelTop smoke-test 1`] = `
<section role="list"
         id="label-top"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels LongValueEnd smoke-test 1`] = `
<section role="list"
         id="long-value-end"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="5,000,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5,000,000
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 10"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 10%;"
      >
      </div>
    </div>
    <label title="500,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        500,000
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 1"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 1%;"
      >
      </div>
    </div>
    <label title="50,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50,000
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: No value"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 0%;"
      >
      </div>
    </div>
    <label title="5,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5,000
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels LongValueStart smoke-test 1`] = `
<section role="list"
         id="long-value-start"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="5,000,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5,000,000
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 10"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 10%;"
      >
      </div>
    </div>
    <label title="500,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        500,000
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 1"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 1%;"
      >
      </div>
    </div>
    <label title="50,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50,000
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: No value"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 0%;"
      >
      </div>
    </div>
    <label title="5,000"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5,000
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels ValueBottom smoke-test 1`] = `
<section role="list"
         id="value-bottom"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueBottom_18e6r_57 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueBottom_18e6r_57 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueBottom_18e6r_57 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueBottom_18e6r_57 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels ValueEnd smoke-test 1`] = `
<section role="list"
         id="value-end"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="200"
           class="_valueLabel_18e6r_71"
    >
      <small>
        200
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 28"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 28%;"
      >
      </div>
    </div>
    <label title="55"
           class="_valueLabel_18e6r_71"
    >
      <small>
        55
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueEnd_18e6r_9 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 3"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 3%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels ValueNone smoke-test 1`] = `
<section role="list"
         id="value-none"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueNone_18e6r_65 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;

exports[`Charts/Bar List/Labels ValueStart smoke-test 1`] = `
<section role="list"
         id="value-start"
         style="width: 350px;"
>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Phishing Attempts"
           class="_label_18e6r_8"
    >
      Phishing Attempts
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Phishing Attempts: 100"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 100%;"
      >
      </div>
    </div>
    <label title="100"
           class="_valueLabel_18e6r_71"
    >
      <small>
        100
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Vulnerability Patch"
           class="_label_18e6r_8"
    >
      Vulnerability Patch
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Vulnerability Patch: 50"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 50%;"
      >
      </div>
    </div>
    <label title="50"
           class="_valueLabel_18e6r_71"
    >
      <small>
        50
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Critical Failure"
           class="_label_18e6r_8"
    >
      Critical Failure
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Critical Failure: 25"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 25%;"
      >
      </div>
    </div>
    <label title="25"
           class="_valueLabel_18e6r_71"
    >
      <small>
        25
      </small>
    </label>
  </div>
  <div role="listitem"
       class="_item_18e6r_1 _valueStart_18e6r_36 _labelTop_18e6r_42"
  >
    <label title="Physical Intrusion"
           class="_label_18e6r_8"
    >
      Physical Intrusion
    </label>
    <div class="_outerBar_18e6r_97">
      <div class="_bar_18e6r_105"
           tabindex="0"
           aria-label="Physical Intrusion: 5"
           role="graphics-document"
           style="background: rgb(41, 181, 202); width: 5%;"
      >
      </div>
    </div>
    <label title="5"
           class="_valueLabel_18e6r_71"
    >
      <small>
        5
      </small>
    </label>
  </div>
</section>
`;
